projects/ngrx.io/content/guide/store/selectors.md

```diff
@@ -294,6 +294,59 @@ export const selectFeatureCount = createSelector(
 );
 </code-example>
 
+## Using Signal Selector
+The `selectSignal` method expects a selector as an input argument and returns a signal of the selected state slice. It has a similar signature to the `select` method, but unlike `select`, `selectSignal` returns a signal instead of an observable.
+
+### Example Usage in Components
+```typescript
+import { Component, inject } from '@angular/core';
+import { NgFor } from '@angular/common';
+import { Store } from '@ngrx/store';
+
+import { selectUsers } from './users.selectors';
+
+@Component({
+  standalone: true,
+  imports: [NgFor],
+  template: `
+    <h1>Users</h1>
+    <ul>
+      <li *ngFor="let user of users()">
+        {{ user.name }}
+      </li>
+    </ul>
+  `
+})
+export class UsersComponent {
+  private readonly store = inject(Store);
+
+  // type: Signal<User[]>
+  readonly users = this.store.selectSignal(selectUsers);
+}
+```
+
+### Selecting with Equality Function
+Similar to the `computed` function, the `selectSignal` method also accepts the equality function to stop the recomputation of the deeper dependency chain if two values are determined to be equal.
+### State Signal in ComponentStore
+The state signal is another addition to the ComponentStore. Instead of using selectSignal, it can be used together with the computed function to create derived signals.
+```typescript
+import { computed, Injectable } from '@angular/core';
+import { ComponentStore } from '@ngrx/component-store';
+
+import { User } from './user.model';
+
+type UsersState = { users: User[]; query: string };
+
+@Injectable()
+export class UsersStore extends ComponentStore<UsersState> {
+  readonly users = computed(() => this.state().users);
+  readonly query = computed(() => this.state().query);
+
+  readonly filteredUsers = computed(() =>
+    this.users().filter(({ name }) => name.includes(this.query()))
+  );
+}
+```
```

Comment:
This can also be removed, because the documentation page is only for `@ngrx/store`, and not for `@ngrx/component-store`.

```suggestion
```